tools/run_tests/sanity/check_submodules.sh

```diff
@@ -31,7 +31,7 @@ cat << EOF | awk '{ print $1 }' | sort > $want_submodules
  886e7d75368e3f4fab3f4d0d3584e4abfc557755 third_party/boringssl-with-bazel (version_for_cocoapods_7.0-857-g886e7d7)
  30dbc81fb5ffdc98ea9b14b1918bfe4e8779b26e third_party/gflags (v2.2.0)
  ec44c6c1675c25b9827aacd08c02433cccde7780 third_party/googletest (release-1.8.0)
- a6189acd18b00611c1dc7042299ad75486f08a1a third_party/protobuf (v3.3.0)
+ 942a29cecd36f2a4b22fdd2179635cd548e6bd27 third_party/protobuf (3.4.x)
```

Comment:
Any idea when the 3.4.0 release is happening?  I know I got some push back when I tried to move the dependency to a non-release protobuf version.